src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -81,6 +101,8 @@ protected String name() {
 
         private final Caller mMutator;
         private final Caller mAccessor;
+        private final WeakHashMap<View, Object> mOriginalValues;
```

Comment:
In this case `WeakHashMap` is special- the weakness of it is important or this class leaks a lot of memory as views disappear and are freed. Honestly, it'd be great if there was a `WeakMap` interface, because the Weak is something we care about but the Hash isn't.